eng/scripts/Language-Settings.ps1

```diff
@@ -88,20 +87,22 @@ function Publish-javascript-GithubIODocs ($DocLocation, $PublicArtifactLocation)
   $PublishedDocs = Get-ChildItem "$($DocLocation)/documentation" | Where-Object -FilterScript { $_.Name.EndsWith(".zip") }
 
   foreach ($Item in $PublishedDocs) 
-  {
-    $PkgName = "azure-$($Item.BaseName)"
-    # Override package name from other scope. We don't have package name to parse this out here.
-    if ($ArtifactToPackgeMap.ContainsKey($($Item.BaseName)))
-    {
-      $PkgName = $ArtifactToPackgeMap[$($Item.BaseName)]
-    }
-    Write-Host $PkgName
+  {    
     Expand-Archive -Force -Path "$($DocLocation)/documentation/$($Item.Name)" -DestinationPath "$($DocLocation)/documentation/$($Item.BaseName)"
     $dirList = Get-ChildItem "$($DocLocation)/documentation/$($Item.BaseName)/$($Item.BaseName)" -Attributes Directory
 
     if ($dirList.Length -eq 1)
     {
       $DocVersion = $dirList[0].Name
+      $parsedPackage = RetrieveReleasePackageInfo "NPM" $PublicArtifactLocation
+      $tag = ""
+      # set default package name
+      $PkgName = "azure-$($Item.BaseName)"
+      if ($parsedPackage -ne $null)
+      {        
+        $PkgName = $parsedPackage.PackageId.Replace("@", "").Replace("/", "-")
+        $tag = $parsedPackage.ReleaseTag
+      }
       Write-Host "Uploading Doc for $($PkgName) Version:- $($DocVersion)..."
       $releaseTag = RetrieveReleaseTag "NPM" $PublicArtifactLocation
```

Comment:
Don't you want to use the tag you got above?

Reply:
I will fetch tag from package info once PR from @chidozieononiwu  is merged